platform/pulp/tasking/tasks.py

```diff
@@ -200,15 +204,16 @@ def apply_async_with_reservation(self, resource_type, resource_id, *args, **kwar
         with transaction.atomic():
             task_status = TaskStatus.objects.create(pk=inner_task_id, state=TaskStatus.WAITING,
                                                     group=group_id, **parent_arg)
-            for tag in tag_list:
+            for tag in tags:
                 task_status.tags.create(name=tag)
 
         # Call the outer task which is a promise to call the real task when it can.
-        _queue_reserved_task.apply_async(args=[task_name, inner_task_id, resource_id, args, kwargs],
+        _queue_reserved_task.apply_async(args=[task_name, inner_task_id, resource_id, args,
```

Comment:
Can this list [task_name, ... ] be converted to a tuple for more memory efficiency?